src/components/task.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Item from './item';
+
+const Task = props => (
+  <li className="task__item">
+    {props.variety !== '' ?
```

Comment:
(Also means you can leave variety out of the default props I think? Since it'll just be undefined? Unless React cries about that.)